docs/website/contents/for-developers/ReleaseProcess.md

```diff
@@ -156,39 +153,40 @@ Suppose we have released the following versions: 1.0.0 and 2.0.0.
 The commit history might look like the following linear chain, with the tags and branch pointers annotated.
 
 ```
-D - the release of FOO-2.0.0 (tag: release-FOO-2.0.0, branch: release-FOO-2.x.x, branch: main)
+D - the release of FOO-2.0.0 (tag: release-FOO-2.0.0, branch: main)
 C - replace that confusing feature with much nicer one
-B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0, branch: release-FOO-1.x.x)
+B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0)
 A - ...
 ```
 
+#### Releasing a new patch version on the main branch
+
 If we subsequently merge a PR that fixes a minor bug present in 2.0.0, it'd then look like this.
 
 ```
 E - fix that off-by-one error (branch: main)
-D - the release of FOO-2.0.0 (tag: release-FOO-2.0.0, branch: release-FOO-2.x.x)
+D - the release of FOO-2.0.0 (tag: release-FOO-2.0.0)
 C - replace that confusing feature with much nicer one
-B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0, branch: release-FOO-1.x.x)
+B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0)
 A - ...
 ```
 
-We could have also advanced the release-FOO-2.x.x branch, but the relevant RULES above do not _require_ doing so until we release version 2.0.1.
-Once we do that, we'd have the following.
+Once we release version 2.0.1, we'd have the following.
 
 ```
-F - the release of FOO-2.0.1 (tag: release-FOO-2.0.1, branch: release-FOO-2.x.x, branch: main)
+F - the release of FOO-2.0.1 (tag: release-FOO-2.0.1, branch: main)
 E - fix that off-by-one error
 D - the release of FOO-2.0.0 (tag: release-FOO-2.0.0)
 C - replace that confusing feature with much nicer one
-B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0, branch: release-FOO-1.x.x)
+B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0)
 A - ...
 ```
 
-We must advance release-FOO-2.x.x to the F commit, because we released F as a version 2.0.X and so the RULE above requires that the F commit is included in the first-parent history of release-FOO-2.x.x.
+#### Releasing a fix exclusive to a previous release
 
 Suppose we then find a bad bug in feature that commit C had replaced.
```

Comment:
Not related to this, but is there something like a "good bug"  (they're called features :smile: ).